### ISSUE TITLE
elm-style CLI

### ISSUE THREAD

**Issue by mdgriffith** [mdgriffith/style-elements-design-discussion#12]
It's be awesome to have some utilities to help manage styles in `style-elements`.

Some features:
  * Detect unused styles.  Prompt to delete them.
  * Maintaining the big "type StyleClass = --{all your classes}" is a pain.  Why not have it be autogenerated/maintained?
  * Generate a new stylesheet file via `elm-style new MyStyleSheet.elm`

**Comment by BrianHicks** [comment]
Hey I'm working on a CLI library. Can we chat about the needs of this tool sometime soon?

**Comment by mdgriffith** [comment]
Yes, yes lets.  I'll ping you tomorrow on slack and we can either discuss there or schedule a call or high five or something :D
